<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserBox</class>
 <widget class="QWidget" name="UserBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>195</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="GroupedToolButton" name="undoButton">
       <property name="toolTip">
        <string>Undo changes by selected user</string>
       </property>
       <property name="icon">
        <iconset theme="edit-undo">
         <normaloff>icons:edit-undo.png</normaloff>icons:edit-undo.png</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="redoButton">
       <property name="toolTip">
        <string>Redo changes by selected user</string>
       </property>
       <property name="icon">
        <iconset theme="edit-redo">
         <normaloff>icons:edit-redo.png</normaloff>icons:edit-redo.png</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="GroupedToolButton" name="opButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Session operator</string>
       </property>
       <property name="icon">
        <iconset theme="irc-operator">
         <normaloff>icons:irc-operator.png</normaloff>icons:irc-operator.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="lockButton">
       <property name="toolTip">
        <string>Lock selected user</string>
       </property>
       <property name="icon">
        <iconset theme="object-locked">
         <normaloff>icons:object-locked.png</normaloff>icons:object-locked.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupCenter</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GroupedToolButton" name="kickButton">
       <property name="toolTip">
        <string>Kick selected user</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff>icons:edit-delete.png</normaloff>icons:edit-delete.png</iconset>
       </property>
       <property name="groupPosition">
        <enum>GroupedToolButton::GroupRight</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="userlist"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupedToolButton</class>
   <extends>QToolButton</extends>
   <header>widgets/groupedtoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
